src/Features/CSharp/Portable/ImplementInterface/CSharpImplementExplicitlyCodeRefactoringProvider.cs

```diff
@@ -90,9 +91,11 @@ protected override bool CheckMemberCanBeConverted(ISymbol member)
                 ? identifierName.Parent
                 : identifierName;
 
+            RoslynDebug.Assert(node is object);
             if (syntaxFacts.IsInvocationExpression(node.Parent))
                 node = node.Parent;
 
+            RoslynDebug.Assert(node is object);
```

Comment:
📝 Not necessary once https://github.com/dotnet/roslyn/pull/40699/files#r362652512 is addressed